vignettes/including-static-dashboards.Rmd

```diff
@@ -0,0 +1,103 @@
+---
+title: "Including Static Dashboards in Crunch"
```

Comment:
Can you also add a `description:` tag? When we build a pkgdown site for crunchy, that will be used in the meta tags for URL previews.